canonical-web-and-design/ubuntu.com
# Build payments summary / auto renewal toggle

## Done

- Added a subscription summary above the table.
- Added a collapsible menu to toggle on and off auto renewal for your monthly subs

## To do

- [ ]  Create the backend endpoint to update auto renewal status


## QA
First you need to add some monthly subscriptions to your account if you don't have some yet:
- Connect to the VPN
- Go here http://10.48.130.194:8080/staging/accounts/
- Connect with your SSO account
- Add some monthly subscriptions to your account

When you have monthly subscriptions:
- Go here https://ubuntu-com-9095.demos.haus/advantage?test_backend=true
- Login with your SSO account
- Check that the summary is correct

## Issue / Card

Fixes #8965 

## Screenshots
![image](https://user-images.githubusercontent.com/11927929/107660310-69b8c180-6c88-11eb-8566-e744f9e0cb4d.png)
